### ISSUE TITLE
Not possible to configure TLS on Jaeger Exporter via SDK auto-configure env vars

### ISSUE THREAD

**Issue by ppatierno** [open-telemetry/opentelemetry-java#4917]
I have a Jaeger backend with TLS enabled on the gRPC port.
Having a look at the Jaeger Exporter source code, it seems to be possible to enable TLS on the gRPC client in order to match the expectations of the backend.
Anyway, I don't see a way to configure TLS on the Jaeger Exporter by using the SDK auto-configure extension through env vars.
Taking a look [here](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md) it seems that for the OTLP exporter there are additional env vars for setting up TLS key and certificates (i.e. `OTEL_EXPORTER_OTLP_TRACES_CERTIFICATE`, `OTEL_EXPORTER_OTLP_TRACES_CLIENT_KEY`, ...).
There are no corresponding env vars for the Jaerger (and Zipkin) exporter.
Is my assumption right that we cannot configure TLS on the Jaeger Exporter client via SDK auto-configuration extension with env vars?

**Comment by jack-berg** [comment]
Unfortunately the environment variables we support are dictated by the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md), and there aren't environment variables for jaeger / zipkin TLS. 

There's been resistance at the spec level to continue to add additional environment variable with knowledge that file based configuration approach has several advantages and is coming down the pipeline. This [issue](https://github.com/open-telemetry/opentelemetry-specification/issues/2891) was opened to shed some light on whether or not new environment variables will be added while we await a file based config solution. Here's the relevant decision from the technical committee: 

> Additionally, new environment variable use-cases will be evaluated and non-essential use-cases may still be blocked on a full configuration specification.

So we're blocked on the spec, and the spec is being cautious about adding new environment variables but may make exceptions in certain situations.

For jaeger, you might consider using an OTLP exporter instead of jaeger since jaeger now [supports OTLP](https://www.jaegertracing.io/docs/1.38/apis/#opentelemetry-protocol-stable). As a matter of fact, there's a [general effort](https://github.com/open-telemetry/opentelemetry-specification/pull/2858) to deprecate and eventually remove the jaeger exporters altogether. The story for [zipkin](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/zipkinexporter) is not as great, but its worth noting that the [collector zipkin exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/zipkinexporter) does support customizing TLS.

**Comment by breedx-splk** [comment]
+1 for what @jack-berg said. I know @ppatierno your question was specifically about auto-configure and env vars, but if you're willing to drop in a little bit of code, the [JaegerGrpcSpanExporterBuilder](https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/jaeger/src/main/java/io/opentelemetry/exporter/jaeger/JaegerGrpcSpanExporterBuilder.java#L108) does support settings certs.  Maybe that helps.

In any case, like Jack said, best bet is to do OTLP into Jaeger at this point.

**Comment by jack-berg** [comment]
Happy to help! Marking this issue as `blocked:spec`. Will close if the spec moves forward with deprecating the jaeger exporter.

**Comment by jack-berg** [comment]
The jaeger exporters were removed in #6119 so this is no longer relevant.
